lib/slack-ruby-bot-server/api/endpoints/teams_endpoint.rb

```diff
@@ -60,6 +61,8 @@ class TeamsEndpoint < Grape::API
               )
             end
 
+            team.state = params[:state] if params[:state]
```

Comment:
This should be `if params.key?(:state)`, because a blank state is different from absence of state.